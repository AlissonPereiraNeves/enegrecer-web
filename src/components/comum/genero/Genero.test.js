import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Genero from '../Genero';

describe('Genero', () => {
  it('deve possuir estrutura definida ', () => {
    const onChange = jest.fn();
    const genero = mount(<Genero id={'13'} onChange={onChange} divClasses={'estado'} />);
    const generoTextoJson = toJson(genero);
    expect(generoTextoJson).toMatchSnapshot();
  });
});