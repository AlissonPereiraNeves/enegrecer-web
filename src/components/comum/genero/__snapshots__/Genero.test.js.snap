// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Genero deve possuir estrutura definida  1`] = `
<campoTexto
  divClasse=""
  id="13"
  inputClasse="validate"
  label="Gênero (máximo de 15 caracteres)"
  maxLen={15}
  placeholder="Ex.: Feminino, Masculino, Não Binário..."
  state="genero"
  type="text"
/>
`;
